cognite/seismic/protos/v1/seismic_service.proto

```diff
@@ -77,6 +77,11 @@ service SeismicAPI {
     **/
     rpc EditSeismicStore (EditSeismicStoreRequest) returns (SeismicStore) {}
 
+    /**
+    Delete a seismic store and associated seismics.
```

Comment:
I think when we delete files, we don't proceed if seismics still exist. Maybe we should match that here to prevent accidentally deleting the wrong seismic store and all its seismics?

Reply:
Makes sense.